katalog/tests/calico/calico.sh

```diff
@@ -14,6 +14,7 @@ load ./../helper
 @test "Install Calico" {
     info
     install() {
+        kubectl apply -f https://raw.githubusercontent.com/sighupio/fury-kubernetes-monitoring/v1.6.0/katalog/prometheus-operator/crd-servicemonitor.yml
```

Comment:
should this be 1.6.1?

Reply:
Hi Ramiro!
Monitoring latest release is 1.6.0.
There are some 1.6.1 release candidates, but not yet. So i will continue with 1.6.0.